### PR TITLE
Implement add and put methods for groups/{id}

### LINKED ISSUE #133: Implement Groups Resource GET (id)
/groups/{id}
- get (return group and users)

### LINKED ISSUE #134: Implement Groups Resource PUT (id)
/groups/{id}
- put (create new model with 2 additional fields: 'added_users', 'deleted_users')

### PR DESCRIPTION
**Pull request by dtymkiv**

### CODE REVIEW

File: src/app/routers/group.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        :param group_id:
+        :return:
+        """
+        group = GroupService.get_by_id(group_id=group_id)
+        if group is None:
+            raise BadRequest("Group is not found")
+        group_json = GroupService.to_json(group, many=False)
+        return jsonify(group_json)
+
+    @login_required
```
Comment (IvanKostiv) [inline src/app/routers/group.py]: extra login required

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/routers/group.py (hunk 1 above)
Comment (Teemmer) [reply src/app/routers/group.py]: Is it ok for any person to view groups?

Comment (Teemmer) [reply src/app/routers/group.py]: Is it ok for any person to view groups?

Comment (Teemmer) [reply src/app/routers/group.py]: Is it ok for any person to view groups?

Comment (Teemmer) [reply src/app/routers/group.py]: Is it ok for any person to view groups?

Comment (Teemmer) [reply src/app/routers/group.py]: Is it ok for any person to view groups?

File: src/app/services/group.py [hunk 2]
```diff
@@ -127,6 +130,9 @@ def get_users_by_group(group_id):
 
         return users
 
+    # @staticmethod
+    # def to_json(data, many=False):
+
```
Comment (IvanKostiv) [inline src/app/services/group.py]: Delete this

**Review by IvanKostiv — DISMISSED** [review]

File: src/app/routers/group.py [hunk 3]
```diff
@@ -88,3 +100,76 @@ def post(self):
...
+    def get(self, group_id):
+        """
+        Get group by id
+
+        :param group_id:
+        :return: Group
+        """
+        group = GroupService.get_by_id(group_id=group_id)
+        if group is None:
+            raise BadRequest("Group is not found")
```
Comment (IvanKostiv) [inline src/app/routers/group.py:131]: BadRequest is status code 400, add to API.doc

**Review by yurdosii — DISMISSED** [review]

**Review by IvanKostiv — APPROVED** [review]

**Review by yurdosii — APPROVED** [review]

**Review by Howrran — APPROVED** [review]

**Review by AndriiKret — APPROVED** [review]

**Pull request merged**